types.cc

```diff
@@ -734,6 +734,14 @@ void write_collection_value(managed_bytes_mutable_view& out, cql_serialization_f
     write_fragmented(out, val);
 }
 
+void write_int32(bytes::iterator& out, int32_t value) {
+    return serialize_int32(out, value);
```

Comment:
why not use serialize_int32() directly in your code? That would make this commit less intrusive, easier to backport, and so on. Especially that the new function is only useful for tests

Reply:
got it, thx